client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -203,11 +203,27 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser> implement
         }
     }
 
+    /**
+     * returns the column definition
+     *
+     * @returns column definition
+     */
     public getColumnDefinition(): string[] {
         const columns = ['name', 'group', 'presence'];
         if (this.isMultiSelect) {
             return ['selector'].concat(columns);
         }
         return columns;
     }
+
+    /**
+     * Sets the user present
+     *
+     * @param viewUser the viewUser Object
+     */
+    public async setPresent(viewUser: ViewUser): Promise<void> {
+        const user = viewUser.user;
+        user.is_present = !viewUser.user.is_present;
```

Comment:
either `viewUser.user.is_present = !viewUser.user.is_present;` and not `const ...` or `user.is_present = !user.is_present;`